ironfish/src/rpc/routes/chain/getBlocks.ts

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { getBlockSize } from '../../../network/utils/serializers'
+import { FullNode } from '../../../node'
+import { GENESIS_BLOCK_SEQUENCE } from '../../../primitives/block'
+import { RpcNotFoundError, RpcValidationError } from '../../adapters'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { RpcBlock, RpcBlockSchema, serializeRpcBlockHeader } from '../types'
+import { RpcTransaction } from './types'
+import { serializeRpcTransaction } from './utils'
+
+const MAX_BLOCKS_RANGE = 30
+
+export type GetBlocksRequest = {
+  /**
+   * The starting block height (inclusive).
+   */
+  start: number
+  /**
+   * The ending block height (exclusive).
+   */
+  end: number
```

Comment:
Our other API's that stream ranges returns the end inclusive, so I think we should keep in line with that.